src/components/views/auth/PasswordLogin.tsx

```diff
@@ -135,7 +166,7 @@ export default class PasswordLogin extends React.Component {
         );
     }
 
-    onUsernameChanged = (ev) => {
+    onUsernameChanged = (ev: any) => {
```

Comment:
`any` is not acceptable during a conversion - it needs to be a defined type, or a comment giving a good reason why the type can't be defined.